core/src/test/java/org/apache/spark/memory/TestMemoryConsumer.java

```diff
@@ -38,12 +38,14 @@ public long spill(long size, MemoryConsumer trigger) throws IOException {
     return used;
   }
 
-  void use(long size) {
+  @VisibleForTesting
```

Comment:
This is a test class, we can just make it public.

Reply:
Ok.